backend/authentication/utils.py

```
@@ -66,7 +66,8 @@ def get_valid_redirect_url(url):
 def blacklist_token(token_str):
     if token_str:
         try:
-            token = RefreshToken(token_str)
+            # We don't verify the token so that, no matter what, we blacklist it.
+            token = RefreshToken(token_str, verify=False)
```

We could also take this out of the try block if we want to make sure that we don't ignore errors for this line.